pkg/minikube/openshiftversions/openshift_versions_test.go

```diff
@@ -112,7 +112,7 @@ func TestGetVersionsMalformed(t *testing.T) {
 
 	_, err := getVersions()
 	if err == nil {
-		t.Fatal("Malformed version value was returned from URL but no error was thrown")
+		t.Fatal("Incorrect version value returned from URL but no error reported.")
```

Comment:
Well, it comes down to semantics again. Malformed URL implies that the URL was wrong and hence something happened which prevented me form retrieving the version information. Your message is more to interpret that I hit an URL which is part of an API and it returned me something, but it was not what I expected. 

So I agree, I also don't want to be pedantic, but there is more to things than just making something sounds good. Given this is a test and the message never being intended to be passed to a user, I'd argue that changing this is not a good idea. I can live with both versions.